src/som/VM.java

```diff
@@ -315,17 +315,43 @@ public void errorExit(final String message) {
     requestExit(1);
   }
 
+  private static class AnsiColor8 {
+    private static String black   = "\u001b[30m";
```

Comment:
Is there a specific reason why these are not final?
and, if they should be final, could we capitalize the names, e.g., `BLACK` to indicate them as constants?

Reply:
It's also getting unwieldily here.
Could you move all printing into something like an `Output` or `Terminal` class?
The `som` package is probably the right place for it.